src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,34 @@
+.card {
```

Comment:
Move to the card file
<img width="835" alt="image" src="https://github.com/user-attachments/assets/c92b64b6-2cb5-4540-b6d1-8f05cf1967bc">